tests/03-auth.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { chai } from 'mochainon';
+import { ResinAuth } from '../lib/auth';
+import { TokenType } from '../lib/token';
+import apiKeyFixtures from './fixtures/api-keys';
+import jwtFixtures from './fixtures/jwts';
+
+const { expect } = chai;
+const IS_BROWSER = typeof window !== 'undefined';
+
+let dataDirectory;
+if (!IS_BROWSER) {
+	// tslint:disable-next-line no-var-requires
+	const settings = require('resin-settings-client');
+	dataDirectory = settings.get('dataDirectory');
+}
+
+const auth = new ResinAuth({ dataDirectory });
+
+describe('ResinAuth', () => {
+	before(() =>
+		// Ensure a clean state before starting
+		auth.removeKey()
```

Comment:
This is a promise - we should return it, so the test doesn't start until the token is definitely removed

Reply:
It is, there are no braces

Reply:
Weird, I could've sworn I checked and saw them... I'm going mad, clearly.